Lib/_strptime.py

```diff
@@ -536,7 +536,7 @@ def _strptime(data_string, format="%a %b %d %H:%M:%S %Y"):
         # the caller didn't supply a year but asked for Feb 29th. We couldn't
         # use the default of 1900 for computations. We set it back to ensure
         # that February 29th is smaller than March 1st.
-        year = 1900
```

Comment:
This is not the right way to fix this. If anything you should just remove the code, but clearly it's here for a reason, so we should discuss why it's here and decide on a course of action in the bug.